homeassistant/components/proxmoxve/manifest.json

```diff
@@ -4,5 +4,5 @@
   "documentation": "https://www.home-assistant.io/integrations/proxmoxve",
   "dependencies": [],
   "codeowners": ["@k4ds3"],
-  "requirements": ["proxmoxer==1.0.3"]
+  "requirements": ["proxmoxer==1.0.3", "requests==2.22.0"]
```

Comment:
Do not add `requests`. That is part of HA itself and it is pinned there.

Reply:
```suggestion
  "requirements": ["proxmoxer==1.0.3"]
```